staging/publishing/rules.yaml

```diff
@@ -4,7 +4,7 @@ recursive-delete-patterns:
 - BUILD.bazel
 - "*/BUILD.bazel"
 - Gopkg.toml
-default-go-version: 1.15.7
+default-go-version: 1.16rc1
```

Comment:
@dims @nikhita -- Have you seen [this](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/98572/pull-publishing-bot-validate/1355249121295863808) crop up w/ pre-release versions before?

```console
 F0129 20:22:31.932516     163 main.go:35] Invalid rules file "/home/prow/go/src/k8s.io/kubernetes/staging/publishing/rules.yaml": validation errors:
- specified go version 1.16rc1 must be a valid go version: Invalid character(s) found in minor number "16rc1" 
```

Reply:
@justaugustus we validate using blang/semver, that's what is failing. See snippet https://play.golang.org/p/gNIlfcIoibb

Reply:
Created https://github.com/kubernetes/publishing-bot/pull/245

Reply:
`pull-publishing-bot-validate` is now green :tada: